### PR TITLE
Adds pizza delivery from another un-named station

### PR DESCRIPTION
**Pull request by optimumtact**
:cl: oranges
add: It looks like a pizza delivery from an un-named station got misplaced
/:cl:

Fixes #20642

### CODE REVIEW

**Comment by Incoming5643** [comment]
https://www.youtube.com/watch?v=DEStdShMRKs

File: code/modules/events/shuttle_loan.dm [hunk 1]
```diff
@@ -60,6 +63,8 @@
 			SSshuttle.centcom_message += "Department resupply incoming."
 		if(ANTIDOTE_NEEDED)
 			SSshuttle.centcom_message += "Virus samples incoming."
+		if(PIZZA_DELIVERY)
+			SSshuttle.centcom_message += "Pizza delivery for Spacestation 13"
```
Comment (PKPenguin321) [inline code/modules/events/shuttle_loan.dm]: Pizza delivery for Spacestation 13****!****

**Review by PKPenguin321 — COMMENTED** [review]
shit i accidentally clicked start review

File: code/modules/events/shuttle_loan.dm [hunk 2]
```diff
@@ -165,6 +170,15 @@
...
 					var/decal = pick(/obj/effect/decal/cleanable/flour, /obj/effect/decal/cleanable/robot_debris, /obj/effect/decal/cleanable/oil)
 					new decal(pick_n_take(empty_shuttle_turfs))
+			if(PIZZA_DELIVERY)
+				shuttle_spawns.Add(/obj/item/pizzabox/margherita)
+				shuttle_spawns.Add(/obj/item/pizzabox/meat)
+				shuttle_spawns.Add(/obj/item/pizzabox/vegetable)
+				shuttle_spawns.Add(/obj/item/pizzabox/margherita)
+				shuttle_spawns.Add(/obj/item/pizzabox/margherita)
+				shuttle_spawns.Add(/obj/item/pizzabox/meat)
+				if(prob(10))
```
Comment (Supermichael777) [inline code/modules/events/shuttle_loan.dm]: do an else with another kind of pizza so the power-gamers can't tell

**Comment by Supermichael777** [comment]
ad an else with another piza so they can't count

oh and make sure its sprite has writing

File: code/modules/events/shuttle_loan.dm [hunk 3]
```diff
@@ -165,6 +170,15 @@
 				for(var/i in 1 to 5)
 					var/decal = pick(/obj/effect/decal/cleanable/flour, /obj/effect/decal/cleanable/robot_debris, /obj/effect/decal/cleanable/oil)
 					new decal(pick_n_take(empty_shuttle_turfs))
+			if(PIZZA_DELIVERY)
```
Comment (Iamgoofball) [inline code/modules/events/shuttle_loan.dm]: should datumize these

**Review by Iamgoofball — CHANGES_REQUESTED** [review]

File: code/modules/events/shuttle_loan.dm [hunk 4]
```diff
@@ -37,6 +38,8 @@
 			bonus_points = 0
 		if(ANTIDOTE_NEEDED)
 			priority_announce("Cargo: Your station has been chosen for an epidemiological research project. Send us your cargo shuttle to receive your research samples.", "Centcom Research Initiatives")
+		if (PIZZA_DELIVERY)
+			priority_announce("Cargo: It looks like a neighbouring station accidentally delivered their pizza to you instead","Centcomm Spacepizza Division")
```
Comment (Cheridan) [inline code/modules/events/shuttle_loan.dm]: Centcom

File: code/modules/events/shuttle_loan.dm (hunk 1 above)
Comment (Cheridan) [inline code/modules/events/shuttle_loan.dm]: Should make it use the station name

File: code/modules/events/shuttle_loan.dm (hunk 4 above)
Comment (ShadowDeath6) [inline code/modules/events/shuttle_loan.dm]: I think it's actuallh centcomm since command has 2 "m"s

File: code/modules/events/shuttle_loan.dm [hunk 5]
```diff
@@ -165,6 +170,15 @@
...
 					new decal(pick_n_take(empty_shuttle_turfs))
+			if(PIZZA_DELIVERY)
+				shuttle_spawns.Add(/obj/item/pizzabox/margherita)
+				shuttle_spawns.Add(/obj/item/pizzabox/meat)
+				shuttle_spawns.Add(/obj/item/pizzabox/vegetable)
+				shuttle_spawns.Add(/obj/item/pizzabox/margherita)
+				shuttle_spawns.Add(/obj/item/pizzabox/margherita)
+				shuttle_spawns.Add(/obj/item/pizzabox/meat)
+				if(prob(10))
+					shuttle_spawns.Add(/obj/item/pizzabox/bomb)
```
Comment (lordpidey) [inline code/modules/events/shuttle_loan.dm]: Small suggestion:  add an else to this prob, that way people can't meta "There's seven boxes!  One of them's a bomb!"

Also, for readability's sake, add identical pizza types in adjacent lines.

File: code/modules/events/shuttle_loan.dm (hunk 4 above)
Comment (Jordie0608) [inline code/modules/events/shuttle_loan.dm]: No it's Centcom, see two lines above this.

**Pull request merged**